CHANGELOG

```diff
@@ -1,6 +1,9 @@
 # 3.20.0 (2025-XX-XX)
 
- * n/a
+ * Bump minimum PHP version to 8.1
```

Comment:
You forgot to update the PHP requirement in composer.json to make Composer aware of this bump.